starter-code/css/main.css

```diff
@@ -6,3 +6,167 @@ Green: #00B172
 White: #FFF
 
 */
+
+.landinghome{
+background-image: url(../images/landing.jpg);
+height: 700px;
+background-size:cover;
+background-repeat: no-repeat;
+}
+
+h2,h3 {
+  color: white;
+  text-align: center;
+  font-family: sans-serif;
```

Comment:
I think the whole page is `sans-serif`. You can apply that property the the whole `body`.